test/fixtures/dialects/spark3/select_cluster_by.sql

```diff
@@ -0,0 +1,10 @@
+-- Produces rows clustered by age. Persons with same age are clustered together.
+-- In the query below, persons with age 18 and 25 are in first partition and the
+-- persons with age 16 are in the second partition. The rows are sorted based
+-- on age within each partition.
+SELECT
+    age,
+    name
+FROM person
+CLUSTER BY
+    age;
```

Comment:
Any chance of a few more examples? Maybe one with delimiters? One with a number? One with an expression?

Reply:
Yup 🙂  Will add in another commit.